appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/BundleDescriptor.java

```diff
@@ -109,7 +106,7 @@
     private boolean keepState = false;
     private boolean defaultGroupPrincipalMapping = true;
 
-    protected HashMap<String, RootXMLNode> rootNodes = new HashMap<String, RootXMLNode>();
+    protected HashMap<String, RootXMLNode> rootNodes = new HashMap<>();
```

Comment:
This comment isn't essential, but a lot of these variables use the implementation as the type, when an interface will probably do